README.md

```diff
@@ -111,6 +111,12 @@ docker build -t nerdctl .
 docker run -it --rm --privileged nerdctl
 ```
 
+### Brew
+On Linux you can install nerdctl via [brew](https://brew.sh):
+```bash
+brew install nerdctl
+```
+
```

Comment:
Can we move this section before “macOS” section, and add a text like “See below for macOS”?

Reply:
Done!